team.html

```diff
@@ -7,23 +7,92 @@ <h1 class="title page-title">Team</h1>
 <p>
     Here are all of the awesome people that make Teach LA happen! We're UCLA students who have dedicated themselves to empowering students through computer science.
 </p>
-<h2 class="title">Board</h2>
+<h2 class="title">General Board</h2>
 
 <p>
     Our directors, school leads, and curriculum leads. Feel free to reach out to any of our board for more information about what we do!
 </p>
 
-{% assign president = site.team | where_exp:"person", "person.group contains 'board' and person.title == 'President'" | group_by:"title" %}
-{% assign board = site.team | where_exp:"item", "item.group contains 'board' and item.title != 'President'" | sort:"name" | group_by:"title" | sort:"name" %}
-{% assign leadership = president | concat: board %}
+<!-- For general board -->
+{% assign president = site.team | where_exp:"person", "person.group contains 'general board' and person.title == 'President'" | group_by:"title" %}
+{% assign vpres = site.team | where_exp: "person", "person.group contains 'general board' and person.title == 'Vice President'" | group_by:"title" %}
+{% assign genBoard = site.team | where_exp:"item", "item.group contains 'general board' and item.title != 'President' and item.title != 'Vice President'" | sort:"name" | group_by:"title" | sort:"name" %}
+{% assign genLeadership = president | concat: vpres | concat: genBoard %}
+
+<!-- For dev board -->
+{% assign devDirector = site.team | where_exp:"person", "person.group contains 'dev board' and person.title == 'Dev Team Director'" | group_by:"title" %}
+{% assign devBoard = site.team | where_exp:"item", "item.group contains 'dev board' and item.title != 'Dev Team Director'" | sort:"name" | group_by:"title" | sort:"name" %}
+{% assign devLeadership = devDirector | concat: devBoard %}
 
 {% assign members = site.team | where_exp:"item", "item.group == 'member'" | sort:"name" %}
 
 {% assign boards = site.team | where_exp:"item", "item.positions" | sort:"name" %}
 {% assign alumsplus = site.team | where_exp:"item", "item.group == 'alum+'" | sort:"name" | group_by:"graduating_year" | sort:"name" %}
 
 <div class="row-desktop-ratio-1-1 text-12x TEAM-SPECIAL">
-    {% for group in leadership %}
+    {% for group in genLeadership %}
```

Comment:
Instead of duplicating code, print according to if they're in genLeadership or devLeadership